### PR TITLE
SQL-1282: add list of unsupported functions to README

### PR DESCRIPTION
**Pull request by jchemburkar**
Added all the function names that call the `unsupported_function!` macro to a table. I felt a bit unsure of the proper way to format, as this read the cleanest, but doubled the length of the README

### CODE REVIEW

File: README.md
Comment (pmeredit) [inline README.md:50]:
```suggestion
The following ODBC functions are currently not supported by the driver. It is possible that some may be supported in future releases. Note that some unsupported ANSI functions have supported Unicode versions. If the version of the function suffixed with 'W', e.g., `SQLTables`, the Unicode version is supported.
```

**Review by pmeredit — APPROVED** [review]
LGTM! I'm not sure if I want my suggestion in there or not. Maybe too much? I'll leave that up to you.

File: README.md
Comment (jchemburkar) [reply README.md:50]: I thought about adding something similar, and agree it might be worthwhile!

**Comment by jchemburkar** [comment]
evergreen merge

**Pull request merged**
